ui_widgets.py

```diff
@@ -173,6 +177,18 @@ def __init__(self, **kwargs):
             highlightcolor=TEXTCOLOR,
         )
 
+        # object sprites
+        self.tank = ImageTk.PhotoImage(Image.open("images/bot2.png"))
```

Comment:
Since we are now reading from the JSON, it doesn't appear these are being used anywhere. As a result, we should probably remove.